shelley-ma/formal-spec/value.tex

```diff
@@ -59,11 +56,11 @@ \section{Coin and Multi-Asset Tokens}
 
 \subsection*{Representing Multi-asset Types and Values}
 An \emph{Asset} comprises a set of different \emph{Asset Classes}, each of which has
-a unique identifier, $\AssetID$. We will informally refer to a pair $(\var{pid}, \var{aid})$ of a Policy ID and an Asset ID as a ``token''. The token for Ada is $(\mathsf{adaID}, \mathsf{adaToken})$.
+a unique identifier, $\AssetName$. We will informally refer to a pair $(\var{pid}, \var{aid})$ of a $\PolicyID$ and an $\AssetName$ as an $\AssetID$. The $\AssetID$ for Ada is $(\mathsf{adaID}, \mathsf{adaToken})$.
 
 The set of tokens that are referred to by the underlying monetary policy represents the coinage that the asset supports.  A multi-asset value, $\Value$ is a map over zero or more assets
 to single asset values.  A single asset value is then a finite map from
```

Comment:
"single asset value" is an odd phrase, which as far as I can tell is subsequently unused.

Reply:
Also, there's a variety of variants on "value" used somewhat inconsistently: "multi-asset value", "single asset value", "asset value", "value", "`Value`", maybe also "token bundle".

Reply:
Yeah, I think the whole paragraph just has to go, and the part about Asset Classes needs a rewrite as well.